client/src/components/OrgMembers.tsx

```diff
@@ -1,13 +1,243 @@
+                                {isOwner ? (<Badge className="custom-owner-badge">Owner</Badge>) : (
+                                    <Badge className="custom-member-badge">Member</Badge>
+                                )}
+                                </div>
+                            </div>
+                            );
+                        })}
+                    </div>
+                </>
+            )}
+
+            {/* Modal for inviting members */}
+            <Modal show={showModal} onHide={handleClose} backdrop="static" centered size="lg">
+                <Modal.Header closeButton>
+                    <Modal.Title>Add Members</Modal.Title>
+                </Modal.Header>
+                <Modal.Body>
+                    <Form.Group className="mb-3">
+                        <Form.Label>Search users by username</Form.Label>
+                        <Form.Control
+                            type="text"
+                            placeholder="Type to search..."
+                            value={searchTerm}
+                            onChange={handleSearchChange}
+                        />
+                    </Form.Group>
+
+                    {searchLoading && <Spinner animation="border" size="sm" />}
+
+                    {searchResults.length > 0 && (
+                        <ListGroup>
+                            {searchResults.filter((user) => !selectedUsers.some((u) => u.id === user.id)).map((user) => (
+                                <ListGroup.Item key={user.id} action onClick={() => toggleSelectUser(user)}>
+                                {user.username} ({user.email})
+                                </ListGroup.Item>
+                            ))}
+                        </ListGroup>
+                    )}
+
+                    {selectedUsers.length > 0 && (
+                        <div className="mt-3">
+                            <strong>Selected users:</strong>
+                            <div className="mt-2 d-flex flex-wrap gap-2">
+                            {selectedUsers.map((u) => (
+                                <span key={u.id} className="chip badge bg-primary text-white d-flex align-items-center"
+                                style={{
+                                    fontSize: "0.9rem",
+                                    padding: "0.5em 0.8em",
+                                    borderRadius: "20px",
+                                    cursor: "pointer",
+                                }} onClick={() => setSelectedUsers(selectedUsers.filter((usr) => usr.id !== u.id))}>
+                                    {u.username}
+                                <i className="bi bi-x-circle ms-2"></i>
+                                </span>
+                            ))}
+                            </div>
+                        <small className="text-muted d-block mt-1">
+                            Click a chip to remove it
+                        </small>
```

Comment:
We don't need this, I think. The `(x)` button on the chip is always visible so it's self-explanatory.